lib/skate_web/auth_manager.ex

```diff
@@ -22,24 +22,15 @@ defmodule SkateWeb.AuthManager do
   def resource_from_claims(_), do: {:error, :invalid_claims}
 
   def username_from_socket!(socket) do
-    {:ok, resource} =
-      socket
-      |> Guardian.Phoenix.Socket.current_token()
-      |> decode_and_verify!()
```

Comment:
We will want to perform `decode_and_verify` when joining a channel - just not as part of getting the username for a resource.